### PR TITLE
adding archetype template files

### PR DESCRIPTION
**Pull request by Poornima545**

### CODE REVIEW

File: archetype/index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            <ul>
+                <li>Home</li>
+                <li>Page layouts</li>
+                <li>Elements</li>
+            </ul>
+        </nav>
+    </header>
+
+
+    <main class="main">
```
Comment (ETulasiram8) [inline archetype/index.html]: sug: avoid taking tag name as a class name

File: archetype/index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                        <li>Col accumsan arcu mi aliquet placerat.</li>
+                        <li>Lobortis vestibulum ut magna tempor massa nascetur.</li>
+                        <li>Blandit massa non blandit tempor interdum.</li>
+                        <li>Lacinia mattis arcu nascetur lobortis.</li>
+                    </ul>
+                </article>
+            </section>
+    </main>
+
+    <footer class="footer">
```
Comment (ETulasiram8) [inline archetype/index.html:88]: sug: same as above comment

File: archetype/style.css [hunk 3]
```diff
@@ -0,0 +1,300 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    line-height: 1.5;
```
Comment (ETulasiram8) [inline archetype/style.css:5]: sug: units missed

File: archetype/style.css [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+    .sidebar p {
+        font-size: 14px;
+    }
+
+    .major {
+        flex-direction: column;
+    }
+
+    .content {
+        order: 2;
```
Comment (ETulasiram8) [inline archetype/style.css:233]: sug: use order :1;

File: archetype/style.css (hunk 3 above)
Comment (Poornima545) [reply archetype/style.css:5]: line height actually don't have any units.

File: archetype/style.css (hunk 4 above)
Comment (Poornima545) [reply archetype/style.css:233]: order 1 didn't work that's why used order 2.

**Pull request merged**